src/config.h

```diff
@@ -130,6 +130,9 @@ struct Config{
     int blob_file_size;
     bool enable_blob_garbage_collection;
     int blob_garbage_collection_age_cutoff;
+    int max_bytes_for_level_base;
+    int max_bytes_for_level_multiplier;
```

Comment:
This configuration item is a `double` in rocksdb, but `int` type in kvrocks, so we should specify it in kvrocks.conf(it would be perfect to support `double`).

Reply:
Yep, I've add date type specify about `max_bytes_for_level_multiplier ` option in `kvrocks.conf`, thanks.